src/components/Layout.tsx

```diff
@@ -10,7 +10,7 @@ import {
   DirectionalHint,
 } from 'office-ui-fabric-react/lib/Tooltip';
 import { getId } from 'office-ui-fabric-react/lib/Utilities';
-import Logo from '../static/img/logo.svg';
+const logo = require('../static/img/logo.svg');
```

Comment:
There is another option like below:

```ts
declare module "*.svg" {
  const content: any;
  export default content;
}
```

But I think writing custom definition is not necessary for this case. Anyway, I will follow your rules. 🤗

## Reference

- [Webpack - Importing Other Assets](https://webpack.js.org/guides/typescript/#importing-other-assets)